### ISSUE TITLE
Screen Reader issue: Contacts headers need to reflect the heading level based on location

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#1989]
Currently all light purple contact headers are being read out as Heading 2.

But, they need to be hierarchical depending on the rest of the page. If the page has tabs, there's an H2 for the tab title, so the contact headers should be H3. Example:
![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/24968ec8-540a-4e7a-ac7c-fd616c2c81f2)![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/be2cf132-6d02-4883-86e8-a97aa9824cce)

If the page doesn't have tabs, there wouldn't be an H2, so the contact headers would be H2. Example:
![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/86524d73-64c8-45f5-9366-fa107f93012f)![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/4f7bd123-1521-4085-bbe3-4b442ceb668a)

**Comment by mcharg** [comment]
@Maroyafaied I'm thinking this one might be able to wait until after launch. It's going to require logic to figure out whether there's a tab when contacts display. It's more important to get the headers reading at all than to not be skipping levels.

**Comment by cthomasONRR** [comment]
@mcharg and I met with Sam and went though the card with the screen reader. He doesn't think this issue should stop us from going live with the new site. Eventually it would be nice to fix but he called this a "first world problem" lol. I'm removing the "required for launch" label.

**Comment by Maroyafaied** [comment]
@R-mohansai I added this issue to this sprint since you started on it. Please let me know when it's ready to be reviewed.

**Comment by R-mohansai** [comment]
Completed and contact now uses tabs as base to shift between h2 and h3. Ready to review.

**Comment by eelzi-ONRR** [comment]
@R-mohansai I'm finding that this works when there is only one level of tabs, but when there is a second level of tabs, the tab heading is H4, and then the contact headers with that read as H3. When the tab is H4, can we get the contact header to be H5? 

Example: [https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=contacts,regional-iimo-contacts](https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=contacts,regional-iimo-contacts)
Scroll down to the Other Contacts and Staff Directory. "Regional IIMO Contacts" is H4, and under that, "Colorado and Other Regions" is H3

**Comment by eelzi-ONRR** [comment]
@R-mohansai 

@cthomasONRR and I have checked the contact headers and accordions in dev and we confirm they have the correct heading tags. Thanks!

**Comment by Maroyafaied** [comment]
@R-mohansai this is not in the live site, once you confirm it's in the live site you can close this issue.
